gateway/testutil.go

```diff
@@ -939,7 +940,8 @@ func (s *Test) Start() {
 		scheme = "https://"
 	}
 	s.URL = scheme + mainProxy().listener.Addr().String()
-
+	// set simple CORS options
+	controlProxy().router.Use(cors.New(cors.Options{}).Handler)
```

Comment:
@urbanishimwe why this change is required?

Reply:
for the server to handle preflight request

Reply:
I've commented this line - and tests are still working

Reply:
me too! removing it